tests/test_qnode.py

```diff
@@ -1094,16 +1094,28 @@ def circuit():
         assert len(circuit.tape.operations) == 2
         assert isinstance(circuit.tape.operations[1], qml.measurements.MidMeasureMP)
 
+    @pytest.mark.parametrize(
+        "dev", [qml.device("default.qubit", wires=3), qml.device("default.qubit.legacy", wires=3)]
```

Comment:
to be clear - these are the same (for now) but will differ very soon

Reply:
Yep, I added it preemptively